cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -492,26 +507,42 @@ protected static Map<Integer, Map<String, Object>> extractLoadParameters(JsonArr
      */
     protected static JsonObject parseLoadStats(JsonArray brokerLoadBeforeArray, JsonArray brokerLoadAfterArray) {
```

Comment:
not strictly related to this PR but it would be good to fix it now, I was wonder if it's better to call these parameters `brokerLoadBeforeJson` and  `brokerLoadAfterArray` which makes more clear these are JSON representation of broker load before and after.

Reply:
Yep, I can fix that